app-i18n/uchardet/uchardet-0.0.6.ebuild

```diff
@@ -7,7 +7,7 @@ inherit cmake-utils
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-SRC_URI="https://www.freedesktop.org/software/${PN}/releases/${P}.tar.xz"
+SRC_URI="https://www.freedesktop.org/software/uchardet/releases/${P}.tar.xz"
```

Comment:
You're not up-to-date on policies. Variables in EGIT_REPO_URI and SRC_URI prefix are strongly discouraged because they make it impossible to test stuff without reinventing bash.